Sources/Networking/Responses/OfferingsResponse.swift

```diff
@@ -18,12 +18,22 @@ import Foundation
 
 public struct PaywallComponentsData: Codable, Equatable, Sendable {
 
+    public struct ComponentsConfigs: Codable, Equatable, Sendable {
+
+        public var base: ComponentsConfig
+
+        public init(base: ComponentsConfig) {
+            self.base = base
+        }
+
+    }
```

Comment:
Added another layer here for something we've been talking about but didn't have any support for. This new `ComponentsConfigs` will allow for multiple paywall configurations:
- Base (the default one and the only required one)
- Landscape
- iPad
- Watch
- Mac

This isn't final at all but wasn't to throw support for this in here because why not 🤷‍♂️

Reply:
Interesting! I guess these are all window dimensions? We're doing window size classes, e.g. defined viewport breakpoints?

Reply:
Very much more long the lines of window size classes! We'll talk about this more at our next team sync 😇  Which means... I should probably extend that meeting time 😅

Reply:
Haha, sweet!